bin/collator/src/cli.rs

```diff
@@ -30,6 +30,10 @@ pub struct Cli {
     #[clap(flatten)]
     pub run: cumulus_client_cli::RunCmd,
 
+    /// Enable Ethereum compatible JSON-RPC servers (disabled by default).
```

Comment:
Can you also please update the zombienet configurations?
Just add this flag to the arguments string.

It's under _Astar/third-party/zombienet_.

Reply:
Added